### ISSUE TITLE
[Masthead] Accessibility checker issue - Element with role "listitem" is not contained in or owned by an element

### ISSUE THREAD

**Issue by rainaabbcc** [carbon-design-system/carbon-for-ibm-dotcom#8485]
### Description

I have a question about dds-masthead component in carbon component , when I use IBM Equal Access Accessibility Checker to scan the page which contains dds-masthead,I found these below violations in this picture , can someone help me to fix it?
![aafe3900-79e2-11ec-806c-e038892a59ab (1)](https://user-images.githubusercontent.com/101535915/158096773-6a061e55-9722-4a63-b0c3-1cdb2e7a7b01.png)

### Component(s) impacted

dds-masthead

### Carbon for IBM.com version

1.24.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

hcdsp

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

 https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main[…]nents/examples/codesandbox/components/footer?file=/index.html

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
@rainaabbcc This issue should be fixed as a part of [1.26.0](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/releases/tag/v1.26.0) in #7284.

Are you able to upgrade to a newer version of this package?

**Comment by rainaabbcc** [comment]
sure,but I have upgraded "@carbon/ibmdotcom-web-components" package to "1.16.1" and it's the The latest version,but also have the problems,and which package should be upgraded?

**Comment by andy-blum** [comment]
@rainaabbcc Can you provide additional context here?

Which elements have the role of listitem?

**Comment by rainaabbcc** [comment]
@andy-blum sorry,Today we use IBM accessibility to scan our page again but the issues disappeared,and when we scan the page in the sandbox the issues also disappeared, thank for your responding,appreciate a lot .
